sdk/src/main/java/com/sygic/travel/sdk/auth/service/AuthService.kt

```diff
@@ -0,0 +1,37 @@
+package com.sygic.travel.sdk.auth.service
+
+import com.sygic.travel.sdk.auth.api.AuthApiClient
+import com.sygic.travel.sdk.auth.model.AuthorizationRequest
+import com.sygic.travel.sdk.auth.model.AuthorizationResponse
+import com.sygic.travel.sdk.auth.model.UserRegistrationRequest
+import com.sygic.travel.sdk.auth.model.UserRegistrationResponse
+import retrofit2.Call
+
+
+class AuthService(
+	private val authClient: AuthApiClient
+) {
+
+	suspend fun authorizationQuery(authRequest: AuthorizationRequest): Call<AuthorizationResponse> {
+		return authClient.authorizationQuery(authRequest)
+	}
+
+	suspend fun userRegistration(name: String, email: String, password: String, accessToken: String)
```

Comment:
coding style, imo the best is:

```kotlin
suspend fun userRegistration(
     name: String, 
     email: String, 
     password: String, 
     accessToken: String
): Call<UserRegistrationResponse> {
}
```

Reply:
- the method name is not good, what about registerUser()
- it's pretty weird to return a call<>, imo there should be already some error handling here.

Reply:
:heavy_check_mark: